playground/test_embedding/test_semantic_search.py

```diff
@@ -0,0 +1,100 @@
+import json
+
+import numpy as np
+import openai
+import pandas as pd
+import requests
+from scipy.spatial.distance import cosine
+
+openai.api_key = "sk-xxx"
```

Comment:
I guess we do not need this. It will automatically read from environment variable `OPENAI_API_KEY`